mathjax3-ts/output/chtml/FontData.ts

```diff
@@ -377,6 +412,36 @@ export class CHTMLFontData extends FontData<CHTMLCharOptions, CHTMLVariantData,
         return em(Math.max(0, n));
     }
 
+    /**
+     * @param {CHTMLCharData} data   The [h, d, w, options] data for the character
+     * @param {number} dw            The (optional) left offset of the glyph
+     * @param {number} ic            The (optional) italic correction value
+     * @return {string}              The padding string for the h, d, w.
+     */
+    public padding(data: CHTMLCharData, dw: number = 0, ic: number = 0) {
```

Comment:
`[h, d, w, options]: CHTMLCharData`?

Reply:
I didn't think of the, either.  I'm not sure I knew it could be done (or if I did, I forgot).